parser.cpp

```diff
@@ -192,12 +196,20 @@ Edl* Parser::parse_body()
             parse_struct_or_union(t == "struct");
         else if (t == "from")
             parse_from_import();
+        else if (t == "#")
+            parse_directive();
         else
         {
             ERROR("unexpected token %s\n", static_cast<std::string>(t).c_str());
         }
     }
 
+    if (!pp_.is_closed())
+    {
+        fprintf(stderr, "unexpected uses of directives.");
```

Comment:
Try to print a message similar to what the C preprocessor would do in case of directives that are not closed.

Reply:
Updated.